currint/tests/test_amount.py

```diff
@@ -2,9 +2,13 @@
 from __future__ import unicode_literals
 from decimal import Decimal
 from unittest import TestCase
+import six
 from ..currency import currencies
 from ..amount import Amount, _ZeroAmount
 
+if six.PY3:
+    from builtins import str as unicode
```

Comment:
I'm still on this project?

You can just use https://pythonhosted.org/six/#six.text_type